core/ppu_bg.go

```diff
@@ -273,12 +314,15 @@ func (ppu *PPU) colorDepth(background uint8) uint8 {
 		switch background {
 		case 0:
 			return 4
+		//TODO offset per tile
+		case 2:
+			return 2
 		default:
 			panic(fmt.Sprintf(panicString, ppu.backgroundData.screenMode, background+1))
 		}
 	case 7:
 		switch background {
-		case 0:
+		case 0, 1:
```

Comment:
same question as before for background 2 in mode 7

Reply:
Yes it can in mode "7EXTBG" which is mode 7 with an extra background layer